src/views/service_panel.dot

```diff
@@ -14,12 +14,12 @@
     {{~ this.categories:category:index }}
       {{?
         !this.mustDeployCategories ||
-        (this.mustDeployCategories && !this.isDeployed && index + 1 < 9) ||
+        (this.mustDeployCategories && !this.isDeployed && index < 12) ||
```

Comment:
I recently found that the whole "deployment" logic is dead code. `isDeployed` is changed only in the `toggleCategories` function, which is never called anywhere. Maybe we can get rid of all that.

Reply:
ok!

Reply:
The whole "deploy" thing was used in an older design, and doesn't exist anymore. I removed it